AlphaWallet/Tokens/Types/TokensDataStore.swift

```diff
@@ -861,3 +867,9 @@ class TokensDataStore {
     }
 }
 // swiftlint:enable type_body_length
+
+extension Realm {
+    var threadSafe: Realm {
+         try! Realm(configuration: self.configuration)
```

Comment:
Is this a slow operation? Or are we ok as long as we create it and batch read/write to it?

Reply:
seems like it ok, but i will try to play with other init method for realm 

`Realm(configuration: <#T##Realm.Configuration#>, queue: <#T##DispatchQueue?#>)
`

to avoid creating threadSafe object